Shared/Samples/Set up location-driven geotriggers/SetUpLocationDrivenGeotriggersView.swift

```diff
@@ -0,0 +1,336 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+import ArcGISToolkit
+
+struct SetUpLocationDrivenGeotriggersView: View {
+    /// The view model for the sample.
+    @StateObject private var model = Model()
+    
+    /// The popup to show the feature information.
+    @State private var popup: Popup?
+    
+    /// A Boolean value indicating whether to show the popup.
+    @State var isShowingPopup = false
+    
+    var body: some View {
+        MapView(map: model.map)
+            .locationDisplay(model.locationDisplay)
+            .task {
+                // Start geotrigger monitoring once the map loads.
+                do {
+                    try await model.map.load()
+                    model.startGeotriggerMonitoring()
+                } catch {
+                    model.error = error
+                }
+            }
+            .overlay(alignment: .top) {
+                // Status text overlay.
+                VStack {
+                    Text("\(model.fenceGeotriggerText)\n")
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                    
+                    Text(model.nearbyFeaturesText)
+                        .foregroundColor(.orange)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .padding(.top, -24)
+                }
+                .padding(8)
+                .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            .toolbar {
+                // Bottom button toolbar.
+                ToolbarItemGroup(placement: .bottomBar) {
+                    Button("Current Section") {
+                        let sectionFeature = model.nearbyFeatures[model.currentSectionName!]!
+                        popup = Popup(geoElement: sectionFeature)
```

Comment:
The old sample supports displaying multiple popups in a page view-like controller. Can you explore if that is possible in [SwiftUI](https://developer.apple.com/documentation/swiftui/tabviewstyle/page)? It's OK if it doesn't.

@mhdostal I vaguely recall we won't support multi-page popup in Swift toolkit, is that correct?

Reply:
I looked into and don't see an easy way to do it. Especially if it isn't supported in the toolkit.

Reply:
> I vaguely recall we won't support multi-page popup in Swift toolkit, is that correct?

Correct, multi-page popups are not supported in the popup in the Swift Toolkit.

Reply:
It works but it super glitchy. The popup kind of spazzes when and wont let you all the way over to the other screen when you try to swipe.